### ISSUE TITLE
Update a11y for Claims Alert & Step Table

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#17734]
## Accessibility Acceptance Criteria

1. Everything on the page should be read by the screen reader.
     - If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)
2. Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap, or horizontal scroll.
3. Each touch target must be at minimum 44px by 44px
4. The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.  
5. All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
6. When a page opens, the first section or element should automatically be placed in focus
7. When a page opens, the title page should be announced

## Accessibility Implementation Guidelines 

**Alert (File request)**
- Treat as static text 

**Step Table**

1. For each item with a dropdown:
Note: the entire item should be tappable and not just the dropdown
- accessibilityLabel= Text
- accessibilityHint="view more details"
- accessibilityRole="spinbutton"
- accessibilityState= Denote when dropdown is expanded or collapsed
- Ensure the heading and date text are grouped together and read together by the screen reader
- Ensure dropdown area does not "trap" the keyboard focus

2. For each expanded item that has a button:
- Mark mobile alert header, e.g."You have 2 file requests from VA", as Header 3
- accessibilityLabel="view file requests"
- accessibilityHint="navigates you to the file upload page"
- accessibilityRole="button"

3. For each item without a dropdown:
- accessibilityLabel= Text

**Alert (w/ Review locations button)**
- accessibilityLabel= "review locations"
- accessibilityHint= "navigates you to the VA claim exam page
- accessibilityRole= "button"

**Comment by digitalken** [comment]
Added to TestRail.

**Comment by digitalken** [comment]
FAIL - 1/5/2021. Logged multiple issue bug.

https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/18160
